src/reducers/ui.js

```diff
@@ -24,8 +24,23 @@ function addNotification(state, type, severity, payload = {}) {
   );
 }
 
+export function reduceRoot(rootState, action) {
+  return rootState.update('ui', (state) => {
+    switch (action.type) {
+      case 'GIST_EXPORT_NOT_DISPLAYED':
```

Comment:
It looks like if there was an existing error message, a subsequent successful export will replace the error message with a success message if we're not able to open a new tab; however if we are able to open a new tab, the old error message is never cleared out. 

Do you think it's worth adding a `case 'GIST_EXPORT_DISPLAYED':` to clear out `state.ui.notifications`?